### ISSUE TITLE
CT: net::ERR_CACHE_READ_FAILURE

### ISSUE THREAD

**Issue by zepumph** [phetsims/aqua#191]
This has been happening a fair bit since https://github.com/phetsims/aqua/commit/8f39ea156daec9273ab99f0ebab230366e7ee4dc. Mostly they seem like false negatives because it 

```
arithmetic : phet-io-studio-fuzz : unbuilt
http://128.138.93.172/continuous-testing/ct-snapshots/1693944541002/studio/?sim=arithmetic&phetioWrapperDebug=true&fuzz&wrapperContinuousTest=%7B%22test%22%3A%5B%22arithmetic%22%2C%22phet-io-studio-fuzz%22%2C%22unbuilt%22%5D%2C%22snapshotName%22%3A%22snapshot-1693944541002%22%2C%22timestamp%22%3A1693945280769%7D
did not load in 60000ms
[URL] http://128.138.93.172/continuous-testing/aqua/html/wrapper-test.html?url=..%2F..%2Fct-snapshots%2F1693944541002%2Fstudio%2F%3Fsim%3Darithmetic%26phetioWrapperDebug%3Dtrue%26fuzz&testInfo=%7B%22test%22%3A%5B%22arithmetic%22%2C%22phet-io-studio-fuzz%22%2C%22unbuilt%22%5D%2C%22snapshotName%22%3A%22snapshot-1693944541002%22%2C%22timestamp%22%3A1693945280769%7D
[NAVIGATED] http://128.138.93.172/continuous-testing/aqua/html/wrapper-test.html?url=..%2F..%2Fct-snapshots%2F1693944541002%2Fstudio%2F%3Fsim%3Darithmetic%26phetioWrapperDebug%3Dtrue%26fuzz&testInfo=%7B%22test%22%3A%5B%22arithmetic%22%2C%22phet-io-studio-fuzz%22%2C%22unbuilt%22%5D%2C%22snapshotName%22%3A%22snapshot-1693944541002%22%2C%22timestamp%22%3A1693945280769%7D
[NAVIGATED] about:blank
[NAVIGATED] http://128.138.93.172/continuous-testing/ct-snapshots/1693944541002/studio/?sim=arithmetic&phetioWrapperDebug=true&fuzz&wrapperContinuousTest=%7B%22test%22%3A%5B%22arithmetic%22%2C%22phet-io-studio-fuzz%22%2C%22unbuilt%22%5D%2C%22snapshotName%22%3A%22snapshot-1693944541002%22%2C%22timestamp%22%3A1693945280769%7D
[NAVIGATED] about:blank
[CONSOLE] enabling assert
[NAVIGATED] about:blank
[NAVIGATED] http://128.138.93.172/continuous-testing/ct-snapshots/1693944541002/arithmetic/arithmetic_en.html?brand=phet-io&ea&debugger&postMessageOnError&sim=arithmetic&phetioWrapperDebug=true&fuzz&wrapperContinuousTest=%7B%22test%22%3A%5B%22arithmetic%22%2C%22phet-io-studio-fuzz%22%2C%22unbuilt%22%5D%2C%22snapshotName%22%3A%22snapshot-1693944541002%22%2C%22timestamp%22%3A1693945280769%7D&phetioEmitAPIBaseline&phetioCreateArchetypes&phetioEmitHighFrequencyEvents=false
[CONSOLE] enabling assert
[CONSOLE] Failed to load resource: net::ERR_CACHE_READ_FAILURE: http://128.138.93.172/continuous-testing/ct-snapshots/1693944541002/chipper/dist/js/sun/js/DefaultSliderTrack.js

```

**Comment by zepumph** [comment]
For now we will bring this back, https://github.com/phetsims/aqua/commit/69fcc4491230bcb74d05f9fc0b20ae68b5916c31

It would be good to bring this back and do some investigating. At the very least we should mention this as a possibility to other devs. "Sometimes green means that along with other passes, it just hasn't loaded and will try again next time."

**Comment by zepumph** [comment]
First step is to see how we can monitor or measure how often CT: net::ERR_CACHE_READ_FAILURE is occurring, in correlation with how many parallel browser processes are running.

**Comment by zepumph** [comment]
This is a large number of the CT reporting errors these days. I'd like to investigate this sooner rather than later.

**Comment by zepumph** [comment]
Perhaps it is internal to puppeteer: https://pptr.dev/api/puppeteer.page.setcacheenabled

**Comment by zepumph** [comment]
Good luck team, and thanks @mattpen for the suggestion.

**Comment by zepumph** [comment]
This worked! We haven't had any of these issues for 2 rows. I feel like it was hiding other issues before, like 502 errors, and this one:

```
[CONSOLE] Access to fetch at 'https://phet-io.colorado.edu/sims/molecule-shapes-basics/1.6.4/molecule-shapes-basics-phet-io-api.json' from origin 'http://128.138.93.172' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource. If an opaque response serves your needs, set the request's mode to 'no-cors' to fetch the resource with CORS disabled.
[CONSOLE] Failed to load resource: net::ERR_FAILED: https://phet-io.colorado.edu/sims/molecule-shapes-basics/1.6.4/molecule-shapes-basics-phet-io-api.json
[PAGE ERROR] Error: TypeError: Failed to fetch
at Client.getAPIFile (http://128.138.93.172/continuous-testing/ct-snapshots/1695328893430/chipper/dist/js/phet-io-wrappers/common/js/Client.js:1561:26)
at Client.createMigrationEngine (http://128.138.93.172/continuous-testing/ct-snapshots/1695328893430/chipper/dist/js/phet-io-wrappers/common/js/Client.js:1274:37)
at async StandardPhetioWrapper.loadStandardPhetioWrapperFromHTML (http://128.138.93.172/continuous-testing/ct-snapshots/1695328893430/chipper/dist/js/phet-io-wrappers/common/js/StandardPhetioWrapper.js:215:29)
at async Client.loadStandardPhetioWrapperFromHTML (http://128.138.93.172/continuous-testing/ct-snapshots/1695328893430/chipper/dist/js/phet-io-wrappers/common/js/Client.js:1348:20)
at async http://128.138.93.172/continuous-testing/ct-snapshots/1695328893430/phet-io-wrappers/migration/migration.js:200:13
[CONSOLE] continuous-test-wrapper-error
```

I'll take a look tomorrow when I can get a bit more trend out of it.

**Comment by zepumph** [comment]
https://github.com/phetsims/aqua/issues/191#issuecomment-1730376686

This still is coming up, but I'll make another issue for that. Closing.
